integrated_channels/canvas/client.py

```diff
@@ -74,29 +78,11 @@ def create_content_metadata(self, serialized_data):
         self._create_session()
 
         # step 1: create the course
-        url = CanvasAPIClient.course_create_endpoint(
-            self.enterprise_configuration.canvas_base_url,
-            self.enterprise_configuration.canvas_account_id
-        )
-        status_code, response_text = self._post(url, serialized_data)
+        status_code, response_text = self._post(self.course_create_url, serialized_data)
+        created_course_id = json.loads(response_text)['id']
```

Comment:
What's the fail safe here? What happens if Canvas returns an error?

Reply:
two cases

[1] if exception thrown here (due to raise_for_status or due to response_text not being valid, for example):
because the base transmitter class is calling this in a try.. except, I believe it will be handled in the 'normal' error handling way (so no transmission records will be created), will that be okay for this round of work? IO was not planning on changing anything there https://github.com/edx/edx-enterprise/blob/f40fe6749e57ef91e6b0f9904700b40b9bb6c651/integrated_channels/integrated_channel/transmitters/content_metadata.py#L127

[2] if no exception here then we just return status_code, response_text

Reply:
discussed offline with Alex, this is all set